docs/release-payload-controllers.md

```diff
@@ -0,0 +1,73 @@
+
+# release-payload-controller
```

Comment:
you are missing a controller (could be pre-existing) that is responsible for setting the prowjobs that need to be monitored by `release-payload-prowjob-controller`